test/parser/JavaMetrics.test.ts

```diff
@@ -211,7 +225,7 @@ describe("Java metrics tests.", () => {
             await testFileMetric(
```

Comment:
The following also applies here:

Let's activate this test and asume that default labels are counted as well.
We agreed that it is ok for now to also count them. So let's relfect that in the tests

_Originally posted by @ce-bo in https://github.com/MaibornWolff/metric-gardener/pull/101#discussion_r1485851071_

Reply:
Since there is a special requirement to count also default statements for now, let's have an individual test for exactly that. Otherwise too much requirements are covered here in the test which makes it hard to understand the implemented behavior.

Reply:
It is now more defined as an "exception" from the normal behaviour that is currently allowed for that language rather than a special requirement that they are counted. 
My Linh has reduced the scope of this test case, I am ok with that now.